src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -931,6 +939,60 @@ private void createAndSetStatistics()
         findPaneOfTypeByID(COOKS_LABEL, Label.class).setLabelText(numberOfcooks);
     }
 
+    /**
+     * Creates several statistics and sets them in the townHall GUI.
+     */
+    private void createAndSetStatistics()
+    {
+        final int citizensSize = townHall.getColony().getCitizens().size();
+
+        Map<String, Integer> jobCountMap = new HashMap<String, Integer>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2293)